CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ### Added
 * **dso-toolkit + sources + css:** CSS Component Application Heading ([#1251](https://github.com/dso-toolkit/dso-toolkit/issues/1251))
+* **css + sources:** CSS Component Context ([#1191](https://github.com/dso-toolkit/dso-toolkit/issues/1191))
```

Comment:
Ik ben het ff kwijt wanneer we nu eigenlijk welk package noemen in de changelog regels. Je hebt het activiteiten voorbeeld aangepast, zou dan niet ook `dso-toolkit` genoemd moeten worden?

Reply:
Done